### ISSUE TITLE
flake8 complains about no definitions for import Actor, keyboard, keys, sounds, clock, music and screen.

### ISSUE THREAD

**Issue by JeremiahCheatham** [lordmauve/pgzero#269]
flake8 and i believe other alternatives will complain that pyzero builtins are not defined.

    from pgzero.builtins import Actor, keyboard, keys, sounds, clock, music

The above seems to be the only work around for most of this. However either changes in stable or maybe was different in current but you can not add screen. In the recent past in current i would add screen but switching back to stable it's not working. So flake8 will constantly complain screen is not defined.

**Comment by ProfessorCode212** [comment]
I agree that this is an issue for me as well. I tried searching for solutions but I have found none. This is quite problematic because it adds a lot of 'noise' and is somewhat distracting. I am using _VS Code_ and the only workaround I have for this is to completely disable the _Python_ extension. It would be great if this issue could be solved soon.

**Comment by ProfessorCode212** [comment]
Okay. I just noticed that a recent commit was made which added instructions for fixing this issue in [`INSTALLATION.rst`](https://github.com/lordmauve/pgzero/blob/c12bdd337767e43f73f3c12fd397e9bda527c47b/doc/installation.rst#for-flake8pyflakes). However, would it be possible to generate a `.flake8` file on the first run of `pgzrun` to avoid this problem altogether?
